lib/src/model.dart

```diff
@@ -178,13 +178,18 @@ abstract class ModelElement {
       return docs;
     }
 
-    String _getMatchingLink(String codeRef) {
+    String _getMatchingLink(String codeRef, [bool isConstructor = false]) {
```

Comment:
sorry, should have caught this the first time. generally, booleans are better has named optional params. It's the difference between:

`foo(true)`

and

`foo(specialFlag: true)`

in readability.

Your call.